sdks/python/apache_beam/typehints/native_type_compatibility_test.py

```diff
@@ -103,6 +98,64 @@ def test_convert_to_beam_types(self):
         native_type_compatibility.convert_to_beam_types(typing_types),
         beam_types)
 
+  def test_is_sub_class(self):
+    self.assertTrue(native_type_compatibility._safe_issubclass(
+        parent=typing.Dict,
+        derived=typing.Dict[bytes, int]))
+    self.assertFalse(native_type_compatibility._safe_issubclass(
+        parent=typing.List,
+        derived=typing.Dict[bytes, int]))
+    self.assertTrue(native_type_compatibility._safe_issubclass(
+        parent=typing.List,
+        derived=typing.List[bytes]))
+    self.assertFalse(native_type_compatibility._safe_issubclass(
```

Comment:
This is a duplicate of line 105.
Also, it is sufficient to focus on different classes of usecases for the function. 
For example between
```
    self.assertTrue(native_type_compatibility._safe_issubclass(
        parent=typing.Set,
        derived=typing.Set[int]))
```
and
```
    self.assertTrue(native_type_compatibility._safe_issubclass(
        parent=typing.Tuple,
        derived=typing.Tuple[int]))
```
I would keep only one scenario.

Reply:
True, removed the duplicate